uaclient/upgrade_lts_contract.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+"""
```

Comment:
Just double checking, what is the rationale for moving this into the uaclient module ?

Reply:
Previously, the reboot_cmds script called `python3 /usr/lib/ubuntu-advantage/upgrade_lts_contract.py`. In the new version, this function is shared in uaclient, and so `reboot_cmds.py` imports the function and calls it instead of starting a subprocess. (If you're going one commit at a time, this happens in a later commit)

Reply:
ohhh indeed. Good point, no blocks on that at all